test/spec/services/applicationGeneratorSpec.js

```diff
@@ -186,6 +186,52 @@ describe("ApplicationGenerator", function(){
             name: "theServiceName"
           },
           host: "www.example.com",
+          wildcardPolicy: "None",
```

Comment:
This is an object the web console is generating, so the value will always be set.